builtin/json.mbt

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 ///|
+#visibility(change_to="readonly", "Use smart constructor to make json")
 pub(all) enum Json {
```

Comment:
two issues I found:
1. #visibility(change_to="readonly") alone would trigger unused warning
2. the IDE does not show the warning